dist/src/main/config/broker.standalone.yaml.template

```
@@ -629,6 +629,12 @@
         # This setting can also be overridden using the environment variable ZEEBE_BROKER_EXPERIMENTAL_RAFT_MAXQUORUMRESPONSETIMEOUT
         # maxQuorumResponseTimeout = 0ms
 
+        # Threshold used by the leader to decide between replicating a snapshot or records.
+        # If a follower lags behind by more than the threshold, the leader will prefer replicating
+        # snapshots.
+        # This setting can also be overridden using the environment variable ZEEBE_BROKER_EXPERIMENTAL_RAFT_PREFERSNAPSHOTREPLICATIONTHRESHOLD.
+        # preferSnapshotReplicationThreshold = 100
```

:+1:

Just a nitpick or idea to make it more clear :shrug: You can decided whether you think it is necessary.
```suggestion
        # Threshold used by the leader to decide between replicating a snapshot or records.
        # If a follower lags behind (with receiving records) by more than the threshold, the leader will prefer replicating
        # snapshots.
        # This setting can also be overridden using the environment variable ZEEBE_BROKER_EXPERIMENTAL_RAFT_PREFERSNAPSHOTREPLICATIONTHRESHOLD.
        # preferSnapshotReplicationThreshold = 100
```

---

I think I'll just reuse what I suggested above (https://github.com/camunda-cloud/zeebe/pull/8562#discussion_r781120303): 
```
        # Threshold used by the leader to decide between replicating a snapshot or records.
        # The unit is number of records by which the follower may lag behind before the leader
        # prefers replicating snapshots instead of records.
        # This setting can also be overridden using the environment variable ZEEBE_BROKER_EXPERIMENTAL_RAFT_PREFERSNAPSHOTREPLICATIONTHRESHOLD.
        # preferSnapshotReplicationThreshold = 100
```